app/facades/station_facade.rb

```diff
@@ -0,0 +1,10 @@
+class StationFacade
+  def self.find_station(location)
+    StationService.find_station(location)
+  end
+
+  def self.specific_station(location)
+    station_data = find_station(location).first
```

Comment:
Careful with something like this (calling `.first`) on a return value. This could error our if no stations were found since you can't find the first element of something that is nil.